.github/workflows/integration-tests.yml

```diff
@@ -28,8 +39,10 @@ jobs:
       - name: Run Integration Tests
         run: ./icingadb-test -icingatesting.debuglog debug.log -test.v
         env:
+          ICINGADB_TESTS_DATABASE_TYPE: ${{ matrix.database.name }}
           ICINGA_TESTING_ICINGADB_BINARY: ${{ github.workspace }}/icingadb
-          ICINGA_TESTING_ICINGADB_SCHEMA: ${{ github.workspace }}/schema/mysql/schema.sql
+          ICINGA_TESTING_ICINGADB_SCHEMA_MYSQL: ${{ github.workspace }}/schema/mysql/schema.sql
+          ICINGA_TESTING_ICINGADB_SCHEMA_PGSQL: ${{ github.workspace }}/schema/pgsql/schema.sql
```

Comment:
Errm... `schema/${{ matrix.database.name }}`?

Reply:
There seems to be no function for converting to uppercase in these expressions. Also I don't know if expressions are even supported in dictionary keys/variable names.

Reply:
For the environment variable name. The code in icinga-testing allows creating both database types within the same test case, so these have to be separate variables.

Reply:
Do we need that?

Reply:
Using the same variable for both the MySQL and PostgreSQL schema import functions wouldn't make code simpler but would break functionality (that we don't need at the moment, to answer you question), just to save one line here.